concent_api/gatekeeper/views.py

```diff
@@ -35,6 +37,10 @@
 @require_POST
 def upload(request):
     logger.debug("Upload request received.")
+
+    if config.SHUTDOWN_MODE is True and not is_request_from_concent_api(request):
+        return JsonResponse({'error': 'Concent is in shutdown mode.'}, status=503)
```

Comment:
Should be extracted to constant, i. e.:
`SHUTDOWN_MODE_ERROR_RESPONSE` or something similar

Reply:
It could and there is a issue (I think) for doing that globally.